THIRDPARTY_LICENSE.txt

```diff
@@ -113,3 +113,16 @@ Copyright 2005 Joshua Tauberer <http://razor.occams.info>
 
 Feel free to use this file however you want, but
 credit would be nice.
+
+*******************************************************************************
+*******************************************************************************
+tweetnacl-js `nacl-fast.js` `nacl-util.js`
```

Comment:
`nacl-util.js` should probably get its own entry as it is not part of the same repository.

Reply:
Done